packages/trpc/server/routers/viewer/eventTypes/getByViewer.handler.ts

```diff
@@ -275,8 +275,7 @@ export const getByViewerHandler = async ({ ctx, input }: GetByViewerOptions) =>
 
   const bookerUrl = await getBookerUrl(user);
   return {
-    // don't display event teams without event types,
-    eventTypeGroups: eventTypeGroups.filter((groupBy) => groupBy.parentId || !!groupBy.eventTypes?.length),
+    eventTypeGroups,
```

Comment:
We now show any event type group even if it is empty, as we show empty screens for them now